src/features/community/components/molecules/CommunityFilter.tsx

```
@@ -0,0 +1,177 @@
+
+import { useAtom } from 'jotai';
+import {
+  Heart,
+  Clock,
+  Album,
+  ArrowDownWideNarrow,
+  ChevronDown,
+  ArrowUpNarrowWide,
+  ALargeSmall,
+  TimerReset,
+  Map,
```

Biome (2.1.2)

[error] Do not shadow the global "Map" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)